governance/program/src/state/realm_config.rs

```diff
@@ -0,0 +1,108 @@
+//! RealmConfig account
+
+use solana_program::{
+    account_info::AccountInfo, program_error::ProgramError, program_pack::IsInitialized,
+    pubkey::Pubkey,
+};
+
+use borsh::{BorshDeserialize, BorshSchema, BorshSerialize};
+
+use crate::{
+    error::GovernanceError,
+    state::enums::GovernanceAccountType,
+    tools::account::{get_account_data, AccountMaxSize},
+};
+
+/// RealmConfig account
+/// The account is an optional extension to RealmConfig stored on Realm account
+#[derive(Clone, Debug, PartialEq, BorshDeserialize, BorshSerialize, BorshSchema)]
+pub struct RealmConfigAccount {
+    /// Governance account type
+    pub account_type: GovernanceAccountType,
+
+    /// The realm the config belong to
+    pub realm: Pubkey,
+
+    /// Addin providing voter weights for community token
+    pub community_voter_weight: Option<Pubkey>,
+
+    /// Reserved for community max vote weight addin
+    pub reserved_1: Option<Pubkey>,
+
+    /// Reserved for council voter weight addin
+    pub reserved_2: Option<Pubkey>,
```

Comment:
should this be called `council_voter_weight_addin` then?

Reply:
Ok, I gave the `addins` concrete names. 
Yes, the common case for council add-in is to have a council without using tokens and with expiry date. This way council members can be removed by the community and they can also be selected periodically through an election process. 
With tokens based votes the members can only voluntarily burn their tokens